$OpenBSD: patch-Scripts_pilfont_py,v 1.3 2010/11/15 20:57:10 sthen Exp $

patching this to use ${MODPY_BIN} is taken care of by
MODPY_ADJ_FILES in the port's Makefile.

--- Scripts/pilfont.py.orig	Sun Nov  1 05:44:12 2009
+++ Scripts/pilfont.py	Sun Nov 14 15:30:26 2010
@@ -1,3 +1,4 @@
+# /usr/local/bin/python
 #
 # The Python Imaging Library
 # $Id: patch-Scripts_pilfont_py,v 1.3 2010/11/15 20:57:10 sthen Exp $
